scripts/make.clj

```diff
@@ -7,6 +8,7 @@
                "clojure-lsp.exe"
                "clojure-lsp"))
 
+#_{:clj-kondo/ignore [:clojure-lsp/unused-public-var]}
```

Comment:
Note: The unused public fn warnings are all false positives since they are referenced from the top level `bb.edn` file.